repair/row.hh

```diff
@@ -71,6 +71,22 @@ public:
         }
         return _fm->representation().size();
     }
+    size_t external_memory_usage() const {
+        size_t size = 0;
+        if (_fm) {
+            size += _fm->representation().size();
+        }
+        if (_dk_with_hash) {
+            size += _dk_with_hash->dk.external_memory_usage();
```

Comment:
The _dk_with_hash is shared by all the rows within the same partition. If we count it for each repair_row, the count is duplicated. We can ignore it.

Reply:
We have to consider the extreme case where there is one row from each partition. This can happen because if the schema has no clustering rows and the content per partition is potentially tiny.

Reply:
We have already considered the partition key size by accounting the _boundary object. 

```
        if (_boundary) {
            size += _boundary->pk.external_memory_usage() + _boundary->position.external_memory_usage();
        }
```